package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cmc-legal-rep-frontend",
-  "version": "1.0.0-alpha",
+  "version": "1.0.0",
```

Comment:
The version number needs to be bumped
Because of how version numbers are compared in RPM, removing alpha would mean this is never greater.
Time to go to 1.1.0 or something.
@sabahirfan This version doesn't map to JIRA, its internal only.

Reply:
reworded to 1.1.0